ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteExporter.java

```diff
@@ -812,12 +807,12 @@ private static void setSwitchCurrentLimit(UcteLine ucteLine, Switch sw) {
             try {
                 ucteLine.setCurrentLimit(Integer.parseInt(sw.getProperty(CURRENT_LIMIT_PROPERTY_KEY)));
             } catch (NumberFormatException exception) {
-                ucteLine.setCurrentLimit((int) DEFAULT_MAX_CURRENT);
-                LOGGER.warn("Switch {}: No current limit, set value to {}", sw.getId(), DEFAULT_MAX_CURRENT);
+                ucteLine.setCurrentLimit(null);
+                LOGGER.warn("Switch {}: No current limit, set value to {}", sw.getId(), null);
```

Comment:
As the current limit is not set, you can replace the message. For example:
No current limit provided